src/categorize.ado

```diff
@@ -11,69 +11,84 @@ Description
 
 __cagegorize__ is a shortcut and extension of {help egen}'s _cut_ function with the icodes option.
 
-Unlike egen cut, __categorize__ does not require the user to include the minumum and the maximum value of the continuous variable in the list of breaks. It creates more descriptive value labels for the generated categorical variable. Users can specify a variable label for the new variable. 
+Unlike egen cut:  
+__categorize__ does not require the user to include the minumum and the maximum value of the continuous variable in the list of breaks.  
+It creates more descriptive value labels for the generated categorical variable.  
+Users can specify a variable label for the new variable.
 
-Finally, users working with age or poverty ratio variables may choose "default" breaks. With default(age), breaks are 18 and 65. With default(povratio), breaks are 50, 100, 150, 200, and 250.
+Finally, users working with age or poverty ratio variables may choose "default" breaks. With _default(age)_, breaks are 18 and 65. With _default(povratio)_, breaks are 50, 100, 150, 200, and 250.
 
 
 Syntax
 ------ 
 
-> __categorize__ _{help newvar}_ =_{help varname}_, [_options_]
+> __categorize__ _{help varname}_, {cmdab:gen:erate}(_{help newvar}_) [_options_]
 
-where _{help newvar}_ is the name of the categorical variable to be generated and _{help varname}_ is the name of the continuous variable in memory.
-
-
-{synoptset 27 tabbed}{...}
+{synoptset 24 tabbed}{...}
 {synopthdr}
 {synoptline}
-    {synopt:{opth breaks(numlist)}}left-hand ends of the grouping intervals. Do not include the minimum or the maximum value of {it:varname}.{p_end}
-	{synopt:{opt default(age|povratio)}}use default breaks. Cannot be combined with the _breaks_ option.{p_end}
+{syntab:Required}
+    {synopt:{opth gen:erate(newvar)}}name of the categorical variable to be generated.{p_end}
+    
+{syntab:Optional}
+    {synopt:{opth breaks(numlist)}}left-hand ends of the grouping intervals. Do not include the minimum or the maximum value of {it:varname}. Either {it:breaks} or {it:default} must be specified.{p_end}
+	{synopt:{opt default(age|povratio)}}use default breaks; cannot be combined with _breaks_.{p_end}
 	{synopt:{opt nolab:el}}{it:newvar} will not be given value labels.{p_end}
 	{synopt:{opt varlab:el(string)}}variable label for _newvar_.{p_end}
 
 
 Example(s)
 ----------
 
-    Generate labelled categorical variable 'income_cat' based on 'pincp'.
-        {bf:. categorize inc_cat = pincp, breaks(15000 30000 50000 100000) varlabel("Income category")}
+    Generate categorical variable 'inc_cat' from 'pincp'.
+        {bf:. categorize pincp, generate(inc_cat) breaks(15000 30000 50000 100000)}
 
-    Generate 'agecat' based on 'agep' with default breaks.
-        {bf:. categorize age_cat = agep, default(age)}
+    Generate 'age_cat' from 'agep' using default breaks.
+        {bf:. categorize age, generate(age_cat) default(age) varlabel("Age group")}
```

Comment:
Change second example to `categorize agep, generate(age_cat) default(age) varlabel("Age group")` so that it works on ACS.